test/controllers/api/v2/repository_sets_controller_test.rb

```diff
@@ -103,6 +103,51 @@ def test_available_repositories
       assert_response :success
     end
 
+    def test_available_repo_sort
+      task = assert_sync_task ::Actions::Katello::RepositorySet::ScanCdn do |product, content_id|
```

Comment:
The primary concern of this test is sorting, so I think we are OK to remove any assertions against this task especially since they are identical to a pre-existing test.

It can be rewritten:

```
task = assert_sync_task ::Actions::Katello::RepositorySet::ScanCdn
```